docs/sandbox_data.md

```diff
@@ -0,0 +1,74 @@
+# Execute this script to populate a MediaFlux server with some sample data:
+# - a metadata schema
+# - a namespace
+# - a collection asset within the namespace
+# - set the collection asset with some values for the fields defined in the metadata schema
+#
+# You can run it from within Aterm with a command as follows:
+#
+# script.execute :in file://full/path/to/tiger-data-app/docs/sandbox_data.md
+
+# ------------------------------------------------------
+# YOU CAN SET THESE VALUES TO YOUR LIKING
+# (The value used for demo_content_file should point to a file that exists on your machine
+# /etc/protocols is a good bet but adust if necessary)
+set demo_schema tigerdata_meta02
+set demo_ns tigerdata_ns02
+set demo_project rdss_project02
+set demo_content_file /etc/protocols
+# ------------------------------------------------------
+
+
+# Fixes the display bug in the Desktop
+actor.grant :type user :name system:manager :role -type role desktop-experimental
+
+
+# Defines a metadata namespace with one document type for "project" data
+asset.doc.namespace.create :namespace $demo_schema :description "TigerData metadata schema"
+
+asset.doc.type.update :create true :description "Project metadata" :type $demo_schema:project :definition < \
+    :element -name id            -type string -index true -min-occurs 1 -max-occurs 1 -label "The unique identifier for the project" \
+    :element -name title         -type string             -min-occurs 1 -max-occurs 1 -label "A plain-language title for the project" \
+    :element -name description   -type string             -min-occurs 1 -max-occurs 1 -label "A brief description of the project" \
+    :element -name data_sponsor  -type string -index true -min-occurs 1 -max-occurs 1 -label "The person who takes primary responsibility for the project" \
+    :element -name data_manager  -type string -index true -min-occurs 1 -max-occurs 1 -label "The person who manages the day-to-day activities for the project" \
+    :element -name data_users_rw -type string -index true -min-occurs 0               -label "A person who has read and write access privileges to the project" \
+    :element -name data_users_ro -type string -index true -min-occurs 0               -label "A person who has read-only access privileges to the project" \
+    :element -name departments   -type string -index true -min-occurs 1               -label "The primary Princeton University department(s) affiliated with the project" \
+    :element -name created_on    -type date               -min-occurs 1 -max-occurs 1 -label "Timestamp project was created" \
+    :element -name created_by    -type string             -min-occurs 1 -max-occurs 1 -label "User that created the project" >
```

Comment:
Uses the pre-MVP fields indicated in Matt's document (https://docs.google.com/document/d/1XjXkzhs5ngM5cmyrmTLaIxG6TF4jT7-HMsMrjj3akRw/edit)